.github/workflows/pr-conventions.yml

```diff
@@ -0,0 +1,38 @@
+name: PR Conventions
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  label-pr:
+    name: Auto-label PR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: netlify/pr-labeler-action@v1.0.0
+        if: startsWith(github.event.pull_request.title, 'fix')
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          label: "type: bug"
+      - uses: netlify/pr-labeler-action@v1.0.0
+        if: startsWith(github.event.pull_request.title, 'chore')
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          label: "type: chore"
+      - uses: netlify/pr-labeler-action@v1.0.0
+        if: startsWith(github.event.pull_request.title, 'feat')
+        with:
+          token: "${{ secrets.GITHUB_TOKEN }}"
+          label: "type: feature"
+
+  lint:
+    name: Lint PR title
```

Comment:
We've disabled this in a lot of the repos, is the impetus here to help with the change log?

Reply:
yeah, commits that don't have this will not show up in the change log.
on other repos we have disabled it because we are not keeping a changelog, but deploying every single merge to main.

Reply:
Github's release management doesn't seem to require special PR titles to generate a changelog. What is the requirement here?

Reply:
sorry i forgot to mention here: release-please uses them to determine if it should bump patch, minor or major.

Reply:
- chore/fix nump patch
- feat bumps minor
- feat/fix/chore with a `!` bump major (breaking changes)

Reply:
i also think it's really useful to have these categories in the changelog